src/main/java/com/gamingmesh/jobs/Placeholders/Placeholder.java

```diff
@@ -37,6 +37,9 @@ public Placeholder(Jobs plugin) {
 	user_furncount,
 	user_maxfurncount,
 	user_doneq,
+    user_dailyquests_pending,
+    user_dailyquests_completed,
+    user_dailyquests_total,

```

Comment:
Remove the white spaces.

Reply:
Wops, sorry, I Hadn't set the tab character yet. Done!